gluoncv/model_zoo/fastscnn.py

```diff
@@ -0,0 +1,344 @@
+
+"""Fast-SCNN, implemented in Gluon."""
```

Comment:
If most of code from fast-seg, you should write one sentence like "Code adapted from xxxx".